frontend/src/data_context_global.tsx

```diff
@@ -121,6 +121,20 @@ export interface Breeding {
   litter_size: number | null;
 }
 
+export interface ExtendedBreeding {
+  [key: string]: any;
+  id?: number;
+  breed_date: string | null;
+  breed_notes: string;
```

Comment:
Are notes mandatory?

Reply:
The mandatory / non-mandatory fields should be reworked. At the moment it's quite messy...

Reply:
yes, I agree. But I'm pretty sure the breed_notes shouldn't be mandatory, so I'll change it.